.gitattributes

```diff
@@ -0,0 +1,14 @@
+* text=auto
```

Comment:
This is cool, didn't know this feature exists. Will take that over into the other branches :)

Reply:
That's a nice way to reduce the size of the shipped applications :+1

Reply:
I will add the tests folder. I just missed it!